lib/tasks/mastodon.rake

```diff
@@ -74,6 +74,13 @@ namespace :mastodon do
       end
     end
   end
+  
+  namespace :users do
+    desc 'clear unconfirmed users'
+    task clear: :environment do
+      User.where('confirmed_at is NULL AND confirmation_sent_at <= ?', 7.days.ago).find_each(&:destroy)
```

Comment:
So, add a line on the first mail or on the web page : "You have two days for validate your account..."